README.md

```diff
@@ -1 +1,52 @@
-# leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> This is a game that sends and receives data from the API
+
+![screenshot](./app_leaderboard.png)
```

Comment:
- [ ] [_OPTIONAL_] Our `README` file is the 1st thing a possible recruiter would puts their attention on when enter our projects, lets make sure we keep it perfect as posible :scroll: :100: 

There is an issue loading the screenshot. :thinking:

Reply:
thanks for the feedback, I've updated README.md